src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements Area, Draw {
+    private String color;
```

Comment:
```suggestion
    protected String color;
```
тоді можна буде в наслідниках викликати поле